content/millennium/r4/encounters/encounter.md

```diff
@@ -56,6 +56,7 @@ _Implementation Notes_
 
 * The [Encounter.hospitalization.destination] will be returned as a reference to a [contained] location resource.
 * The [Encounter.location.location] may be returned as a reference to a [contained] location resource.
+* A populated Encounter response bundle will be sorted based on the start of the [Encounter.period].
```

Comment:
Could we maybe clarify this a little bit more? Is it oldest -> newest or newest -> oldest?

Reply:
https://github.com/cerner/fhir.cerner.com/pull/381/commits/bfe93e45d4e2ea096da8bf9e80782f015c54bdc1

Reply:
<img width="652" alt="Screen Shot 2020-06-12 at 8 27 27 AM" src="https://user-images.githubusercontent.com/19785562/84507526-94c8bc80-ac86-11ea-9fab-48dc54dc8a43.png">